src/math/lp/static_matrix.h

```diff
@@ -79,7 +79,7 @@ class static_matrix
         ref(static_matrix & m, unsigned row, unsigned col):m_matrix(m), m_row(row), m_col(col) {}
         ref & operator=(T const & v) { m_matrix.set( m_row, m_col, v); return *this; }
 
-        ref operator=(ref & v) { m_matrix.set(m_row, m_col, v.m_matrix.get(v.m_row, v.m_col)); return *this; }
+        ref operator=(ref & v) { m_matrix.set(m_row, m_col, v.m_matrix.get_val(v.m_row, v.m_col)); return *this; }
```

Comment:
Is this some javascript shit that needed to be fixed?

Reply:
These are regular compiler errors. But they don't show up during a regular build (maybe it has something to do with the emscripten, which is used for the building or the fact that we are using a static library build).

Reply:
That's really weird. Is this file included anywhere? Maybe it is not used but emscripten takes everything, even files that are not used.

Reply:
Yes, I have a similar feeling. It is included somewhere but it does not mean it is really used. Maybe this was a part of a dead code.